src/main/java/org/prebid/server/bidder/ix/IxBidder.java

```diff
@@ -132,6 +136,22 @@ private ExtImpIx parseImpExt(Imp imp) {
         return modifiedBidRequests;
     }
 
+    /**
+     * IndexExchange does nothing if `buyerid` is not present on the request.
+     * This methods adds the default `0` buyerid. Maybe IX will add the IX UID to this adapter in the future.
+     *
+     * @param user user object for the current bid request
+     * @return the user with a default value for buyerid
+     * @see [https://kb.indexexchange.com/publishers/openrtb_integration/list_of_supported_openrtb_bid_request_fields_for_sellers.htm#User]
+     */
+    private static User modifyUser(User user) {
+        return user == null
+                ? User.builder().buyeruid("0").build()
+                : user.toBuilder()
+                .buyeruid(ObjectUtils.defaultIfNull(user.getBuyeruid(), "0"))
```

Comment:
Minor. Should we use StringUtils.defaultIfEmpty()?

Reply:
The IX docs state that an empty string is fine as well